packages/tx/src/eip2930Transaction.ts

```diff
@@ -138,7 +145,7 @@ export class AccessListEIP2930Transaction extends BaseTransaction<AccessListEIP2
    * the static factory methods to assist in creating a Transaction object from
    * varying data types.
    */
-  public constructor(txData: AccessListEIP2930TxData, opts: TxOptions = {}) {
+  public constructor(txData: TxData[TransactionType.AccessListEIP2930], opts: TxOptions = {}) {
```

Comment:
I would like to define all ` ...[TransactionType.AccessListEIP2930]` references on top of each transaction class file as:
```
type TxData = AllTypesTxData[TransactionType.AccessListEIP2930]
```
and then use just `TxData`, `TxValuesArray` etc etc in the class def for brevity

Reply:
Good idea, done!